packages/manager/apps/web-domains/src/alldoms/pages/service/serviceList/serviceList.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import '@/alldoms/setupTests';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render } from '@testing-library/react';
+import React, { PropsWithChildren } from 'react';
+import { vi } from 'vitest';
+import ServiceList from './serviceList';
+
+vi.mock('react-router-dom', () => ({
```

Comment:
prefer putting those in test setup to avoid having to duplicate it in each test files